### PR TITLE
removes changeling last resort

### PR DESCRIPTION
**Pull request by Spacedong**
because adrenals and having common sense is enough to save you, we don't need an explosion that blinds everyone on the screen that can see you regardless of their gear and spawning a simple mob that can travel at the speed of light

### CODE REVIEW

**Comment by trifornt-tinfornt** [comment]
Not to mention literally free lives.
do it
destroy ling

**Comment by McBawbaggings** [comment]
>No changelog

Add that please.
But yeah fuck last resort as an ability in general.

**Comment by Spacedong** [comment]
why'd travis barf up a failure

**Comment by Swagile** [comment]
why remove entire portions of code when you can simply just comment it out?

that way if we have to revert, you can just remove the comments and walla, code done

for small code like i did its not really needed since you only need to change some variables, but if your removing huge swathes of code like this, i would assume commenting it out is better practice

**Comment by trifornt-tinfornt** [comment]
yeah comment it out instead

**Comment by Spacedong** [comment]
for what purpose travis

**Comment by gumner** [comment]
erradicate lings

**Comment by JohnGinnane** [comment]
I think your code is buggered

File: code/game/gamemodes/changeling/powers/headcrab.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/obj/effect/proc_holder/changeling/headcrab
+/*/ /obj/effect/proc_holder/changeling/headcrab
```
Comment (JohnGinnane) [inline code/game/gamemodes/changeling/powers/headcrab.dm]: I think you just need "/* ..." not "/ /* ..."

**Comment by lcass** [comment]
replacing it with a weaker ability might be better than just scrapping it entirely although it certainly needs changing.

**Comment by Kierany9** [comment]
hey you know monkey form? an ability used to get you out of a jam? lets make a better version of that that stuns everybody on the screen when you activate it.

**Comment by Pyko1** [comment]
Why not just make the lastresort ling turn in to something like alien larva, so it needs to hide.
And instead of being vicious thinggy, it can slowly take over a dead/restrained human/monkey.

**Comment by Sarcalogo** [comment]
How the fuck is this grudge code you shitty fucking maintainers? This shit used to be a fucking TIER 3 POWER. And /tg/ STILL removed it for being retardedly OP. Now changelings get it for FUCKING FREE. You are a fucking retard. Remove this shit.

**Comment by Spacedong** [comment]
Remove the Saltdong, Grudgecode, and DNM labels. This needs to be either approved or denied by the headmins because this ability has gotten shit flung at it for too long and the concept of it is just not fun at all.

**Comment by ArcLumin** [comment]
The DNM was before it was fixed, but now that it's fixed I'll remove the DNM. But, it is grudgecode (I mean, I get it, last resort is broken and should be buffed/removed), and you are pretty salty about it, so I'll leave those.

At least it's not 50 venom death stings :^)

Also, I support this change. Shit needs a nerf, and chopping off ling heads is a good way to deal with them and nerf em.

**Comment by Swagile** [comment]
yeah this is no fun at all

its also an ability that makes it so that even if you SOMEHOW manage to discover a ling your fucked out of your kill by a AoE KO stun and having to fight a small sprite that can instantly vent crawl

GM +1 vote for getting this merged as long as code checks out

**Comment by ArcLumin** [comment]
Code's good. I mean, it's just commenting out, but hey.

**Comment by Hackole** [comment]
This was added in to changelings by default for a reason. Delimbing allowed you to completely kill a changeling by just decapitation - until this was added. Nerf, don't remove. -1

**Comment by SpankMasterson** [comment]
Maybe change it so there is not AoE stun and instead of a ventcrawling killing machine it literally becomes a very slow moving slug that could maybe just like hide under tables

**Comment by Swagile** [comment]
yeah make it like larvae code where they can hide, but can't vent crawl INSTANTLY

and remove AOE KO stun, its a LAST RESORT for a reason, not a free stun and get out of jail free card / kill on any low HP player then egg'd

**Comment by Hackole** [comment]
That'd be a really shitty solution

**Comment by Sarcalogo** [comment]
How?

**Comment by KittyKathy** [comment]
How about this they can only use Last Resort when they are actually beheaded not whenever they want.
When a ling gets beheaded and uses this ability it does not stun it just creates the mob from the head this is how it works in the game now, the headslug also spawns outside of lockers and such.
They can also probably infect their old body from which they were beheaded.
Can you use this ability as a brain right now? I mean right now in game can you behead a ling quickly then debrain the severed head quickly enough to stop them from using last resort? If you are extra paranoid you would acid the brain on the spot to destroy it or maybe acid the head skipping the debraining part entirely.

**Comment by JohnGinnane** [comment]
Crematorium will be even more popular

**Comment by MethMedic** [comment]
Last resort sucks, leave it in. Almost everyone that last resorts ends up dying or ghosting

**Comment by Kierany9** [comment]
>Last resort sucks

no

**Comment by doobiedoo23** [comment]
Last resort doesn't suck meth, jesus

**Comment by Spacedong** [comment]
can we get opinions on this

**Comment by Swagile** [comment]
ive already given my +1 if this works

**Comment by Sarcalogo** [comment]
Kitty Kathy is right, I didn't think through the not being able to be decapitated when regenerating, it's an antag check. That's the solution, and I'm pretty sure I suggested it long before they did in the original pull request by Crystalwarrior.

**Comment by Gandalf2k15** [comment]
This needs head input still. Can't go anywhere without it.

**Comment by McBawbaggings** [comment]
I can see where hackole is coming from.
If they can't vent crawl as to prevent easy escapes while stunned then I'd be fine with with last resort.
The stun is awful but the mob itself isn't a huge threat, I can see this when a ling is about to die but almost killed their target to turn the fight into their favour. Their ability to keep escaping as easily as it currently is the issue imo.

**Comment by ArcLumin** [comment]
Remove vent crawl, give them blood crawl

There is no way this could end poorly

**Comment by Pyko1** [comment]
Just make it a whole lot harder to get back in to the round for em and keep vent crawl.

Ie. they have to find non-husked corpse or someone who is buckled and cuffed, and then spend like 30 seconds burrowing in their brain.

**Comment by Pyko1** [comment]
Or if last resort is removed there should be at least some kind of headspider thinggy crap for lings that get their head randomly chopped off in a middle of hallway and no one even bother to dispose the parts.

**Comment by ArcLumin** [comment]
Basic changes:

1. No stun on activation

2. Longer time between hitting the button and it happening (I.e. the head begins to shake!)

3. Planting a ling egg in someone is channeling required, so they need to stay still

**Comment by GuyonBroadway** [comment]
I think a brief blind is fine so the headslug has a chance to scurry away.

**Comment by yobob591** [comment]
Yeah broadcasting that they are about to release a headslug plus no stun or blind means super easy kill. Should blind them with blood for a second or so

**Comment by McBawbaggings** [comment]
I think we should encourage to use the headslug as a last resort to finish the guy you're fighting rather than the stun and escape mechanism it is now. The stun should stay to give the chance of the headslug to finish biting down the target since they only do 5 damage a hit.

**Comment by GuyonBroadway** [comment]
I disagree as it would be annoying to have a guy bang to rights only for the last resort to pop out and him kill you. At least if it runs away you have a chance to track it down and finish it off. Perhaps consider adding a way to check if a corpse has been infected with a headslug egg, something like a %20 chance on shining a penlight in its eyes for a message to pop up that says something like "Their eyes are strangely watery for a corpse." The % chance is to prevent someone quickly just checking every body and making headslug worthless but if you are pretty sure its infected that body you can preform a check to confirm before burning the corpse.

**Comment by McBawbaggings** [comment]
The idea behind my decision isn't as a guaranteed kill against your assailant but rather if you bought fought to the death and you _almost_ killed him then you would be able to finish him off and use his body as your own.

**Comment by ArcLumin** [comment]
You can already do a quick corpse check. Just spam attack the corpse with a sharp weapon till the organs spill out. Grab the egg.

**Comment by blobbernaut** [comment]
adrenals are fucking removed, close this

**Comment by yobob591** [comment]
what does that have to do with anythinf

**Comment by Swagile** [comment]
this is still a shit game mechanic adrenalines or not because any decent player playing changeling can keep fucking you and everyone over and if they finally get killed, they can re-do it all again because of this mechanic

catching a changeling in the act (RARE AS FUCK) or when its murderboning should end it all, not just restart because of a magical AOE stun + hard to hit sprite + instant vent crawl + instant egging

**Comment by blobbernaut** [comment]
But decapitation should not kill them 

Why exactly should those things mean they just die? They're intended to be slippery and resiliant.

**Comment by blobbernaut** [comment]
If they can regenerate from a head then removing this is fine

**Comment by yobob591** [comment]
another idea, make it your head into a literal head crab that behaves like a drone if you are decapitated and you can find your corpse/decapitate a corpse and reattach your head to it

**Comment by Sarcalogo** [comment]
>Maintainers in charge of merging or closing pull requests.

**Comment by ArcLumin** [comment]
There isn't two head denials or approvals, so not closing it ir merging it.

**Comment by Sarcalogo** [comment]
Well, get the third person on, this is like 20 days old.

**Comment by ArcLumin** [comment]
The general consensus is that this shouldn't be removed, but reworked. Seeing as this PR is only for removal, closing

**Pull request closed without merging**